CMSC320FinalProj_ParkEllis.ipynb

```diff
@@ -806,7 +868,7 @@
     "plt.ylabel('Combined 6x6 & 7x7')\n",
     "plt.title('Average rank of 4x4 and 5x5 vs. 6x6 and 7x7')\n",
```

Comment:
For consistency, we should use only "4x4 & 5x5" or "4x4 and 5x5". I prefer using "&" in this case because "and" is a conjunction that can be used in sentences whereas "&" isn't really used that way, so "&" is less ambiguous when used in sentences. For example, "compare 4x4 and 5x5 and 6x6 and 7x7" is quite confusing